tools/run_tests/run_tests.py

```diff
@@ -483,24 +483,24 @@ def _compiler_options(self, use_docker, compiler):
             return ('jessie', [])
         elif compiler == 'gcc5.3':
             return ('ubuntu1604', [])
-        elif compiler == 'gcc7.4':
-            return ('ubuntu1804', [])
         elif compiler == 'gcc8.3':
             return ('buster', [])
         elif compiler == 'gcc8.3_openssl102':
             return ('buster_openssl102', [
                 "-DgRPC_SSL_PROVIDER=package",
             ])
+        elif compiler == 'gcc11':
+            return ('gcc_11', [])
         elif compiler == 'gcc_musl':
             return ('alpine', [])
-        elif compiler == 'clang4.0':
-            return ('ubuntu1604',
+        elif compiler == 'clang3.7':
```

Comment:
the PR description doesn't mention what's the benefit of replacing clang4.0 with clang3.7

Reply:
Oh this PR isn't in draft. I chose the wrong button. I'll update the description.